examples/demos/devices_users_demo/main.go

```diff
@@ -42,28 +42,28 @@ func createDevice(client *kentikapi.Client) models.ID {
 		"testapi_dns_awssubnet",
 		models.DeviceSubtypeAwsSubnet,
 		1,
-		models.ID(11466),
+		models.ID("11466"),
 		models.CDNAttributeYes,
 	)
-	device.SiteID = models.IDPtr(8483)
+	device.SiteID = models.IDPtr("8483")
```

Comment:
Im wondering if we should rename IDPtr to OptionalID; the fact that we are using pointer is just a technical workaround for Go missing Optionals, and the information our interface should communicate is that the ID is optional
?

Reply:
So  `device.SiteID = models.IDPtr("8483")` would look like `device.SiteID = models.OptionalID("8483")`
I have no preference here.

Reply:
I would go for OptionalID to make things explicit